navigation/build.gradle

```diff
@@ -47,6 +47,10 @@ dependencies {
     api "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleVersion"
     api "androidx.lifecycle:lifecycle-common-java8:$lifecycleVersion"
 
+    // Publishes the lint checks from the referenced module into this library.
+    // Consumers of 'safe.navigation' will have these bundled lint checks applied.
+    lintPublish project(":navigation-lint")
```

Comment:
wasn't aware of this "lintPublish" nice, seems to be working as expected. Could you update the readme and update the version and then I think it's ready to merge.